tests/test_enterprise/api/test_views.py

```diff
@@ -3423,7 +3425,7 @@ class TestBulkEnrollment(BaseTestEnterpriseAPIViews):
     )
     @ddt.unpack
     @mock.patch('enterprise.api.v1.views.get_best_mode_from_course_key')
-    @mock.patch('enterprise.api.v1.views.track_enrollment')
+    @mock.patch('enterprise.utils.track_enrollment')
```

Comment:
I think this change right here is why your test is failing @binodpant

Reply:
yes thanks this was def an issue too